contribs/gmf/examples/mobilequery.js

```diff
@@ -0,0 +1,137 @@
+goog.provide('gmf-mobilequery');
+
+goog.require('gmf.QueryManager');
+goog.require('gmf.Themes');
+goog.require('gmf.layertreeDirective');
+goog.require('gmf.mapDirective');
+goog.require('ngeo');
+goog.require('ngeo.Query');
+goog.require('ngeo.btnDirective');
+goog.require('ngeo.mobileQueryDirective');
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.proj');
+goog.require('ol.source.OSM');
+
+
+proj4.defs('EPSG:21781',
+    '+proj=somerc +lat_0=46.95240555555556 +lon_0=7.439583333333333 +k_0=1 ' +
+    '+x_0=600000 +y_0=200000 +ellps=bessel ' +
+    '+towgs84=674.374,15.056,405.346,0,0,0,0 +units=m +no_defs');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['gmf']);
+
+
+app.module.constant('ngeoQueryOptions', {
+  'limit': 20
+});
+
+
+app.module.constant(
+    'gmfTreeUrl',
+    'https://geomapfish-demo.camptocamp.net/2.0/wsgi/themes?' +
+        'version=2&background=background');
+
+
+app.module.value('gmfWmsUrl',
+    'https://geomapfish-demo.camptocamp.net/1.6/wsgi/mapserv_proxy');
+
+
+/**
+ * A sample directive to display the result.
+ *
+ * @return {angular.Directive} The directive specs.
+ */
+app.queryresultDirective = function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    controller: 'AppQueryresultController',
+    controllerAs: 'qrCtrl',
+    bindToController: true,
+    templateUrl: 'partials/queryresult.html'
+  };
+};
+
+app.module.directive('appQueryresult', app.queryresultDirective);
+
+
+
+/**
+ * @param {ngeo.QueryResult} ngeoQueryResult Query service.
+ * @constructor
+ * @ngInject
+ */
+app.QueryresultController = function(ngeoQueryResult) {
+
+  /**
+   * @type {ngeo.QueryResult}
+   * @export
+   */
+  this.result = ngeoQueryResult;
+
+};
+
+
+app.module.controller('AppQueryresultController', app.QueryresultController);
+
+
+
+/**
+ * @constructor
+ * @param {gmf.Themes} gmfThemes The gme themes service.
+ * @param {gmf.QueryManager} gmfQueryManager The gmf query manager service.
+ */
+app.MainController = function(gmfThemes, gmfQueryManager) {
```

Comment:
It's required here.  If we don't inject this, then it's not created and it doesn't do its magic.